### PR TITLE
mastodon

### PR DESCRIPTION
**Pull request by nfbarrett**
added mastodon support. This is my first pull request. Let me know if I did something wrong.

### CODE REVIEW

File: icons/mastodon.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="61.076954mm" height="65.47831mm" viewBox="0 0 216.4144 232.00976">
```
Comment (dschep) [inline icons/mastodon.svg]: Could you make this icon black & white and the same size as the others? see fb for an example: https://github.com/dschep/proven/blob/master/icons/facebook.svg

**Comment by dschep** [comment]
So, I tested this and it doesn't seem to quite work. 
image

It seems the string varies by mastodon provider, and I don't see any reliable way of identifying a proof type as a mastodon proof. Here are just a few of the available instances to proove yourself on: 
image

changing https://github.com/dschep/proven/pull/30/files#diff-7f0d6418aa692eba917bad748c670f71R52 to `['mastodon.technology']` instead of `mastodon` made it work for @arusahni's proof, but anyone on another instance won't work.

I'm happy to accept a version of this that adds all of the instances currently available, but I don't know how sustainable a model that will be. It'd be nice if @keybase provided a way to get icons for proof types. cc @maxtaco @malgorithms

**Comment by mlsteele** [comment]
Hi, I work at Keybase. You're right, Mastodon instances are part of a [new wave](https://keybase.io/blog/keybase-proofs-for-mastodon-and-everyone) of generic proof services.

To see the ever-growing list of proof services try this:
```
curl "https://keybase.io/_/api/1.0/merkle/proof_params.json?hash=$(curl https://keybase.io/_/api/1.0/merkle/root.json | jq '.payload_json | fromjson .body.proof_services_hash' -r)" | jq '.kit_json | fromjson' | less
```
There's a lot in there, but for icons just look at `.display.key` and `.display.logo_key`.

Images for each proof service are available at this path.
https://keybase.io/images/paramproofs/services/dmoj.ca/logo_full_64@2x.png
Where `dmoj.ca` is the name of the service like `mastodon.social` or `sunbeam.city`. Our client uses these assets. So while it's not exactly part of a public api, I see no reason for us to move them. Just keep in mind the caveat that for services with a `logo_key`, use that instead as an alias.

The sizes available are:
```
logo_black_16.png
logo_black_16@2x.png
logo_full_192.png
logo_full_192@2x.png
logo_full_32.png
logo_full_32@2x.png
logo_full_64.png
logo_full_64@2x.png
```

Hope that helps!

**Comment by dschep** [comment]
Awesome!! thanks @mlsteele!

**Pull request merged**